artifacts/pom.xml

```diff
@@ -34,27 +34,27 @@
 		</developer>
 	</developers>
 
-    <repositories>
```

Comment:
These whitespaces are coming because post the conflict, we had to put the file changes manually which should be fine because there is no indentation issue.